functions/Copy-DbaDatabase.ps1

```diff
@@ -995,10 +995,17 @@ function Copy-DbaDatabase {
 					}
 
 					$dbFinish = Get-Date
-
 					if ($NoRecovery -eq $false) {
+						$dbOwner = $sourceServer.Databases[$dbName].Owner
+						if ($dbOwner -eq $null) {
+							$dbOwner = Get-SaLoginName -SqlInstance $destServer
+						}
 						Write-Message -Level Verbose -Message "Updating database owner"
-						$result = Update-Sqldbowner -source $sourceServer -destination $destServer -dbname $dbName
+						try {
+							$null = Set-DbaDatabaseOwner -SqlInstance $destServer -Database $dbName -TargetLogin $dbOwner -Silent
+						} catch {
+							Write-Message -Level Warning -Message "Failed to update database owner"
```

Comment:
These needs to be updated to output the error, something that will be done for the command as a whole but in this situation it shows why it fails to update the owner.

The `Set-DbaDatabaseOwner` has not been updated to include the Silent parameter.